src/PrestaShopBundle/Controller/Admin/FrameworkBundleAdminController.php

```diff
@@ -526,4 +528,93 @@ protected function getErrorMessageForException(Exception $e, array $messages)
             $e->getMessage()
         );
     }
+
+    /**
+     * Adds a new stylesheet(s) to the page header.
+     *
+     * Warning: this implementation will be broken when layout is migrated
+     * from legacy to Symfony as it relies on the layout being managed
+     * by a legacy controller
+     *
+     * @param string|array $cssUri Path to CSS file, or list of css files like this : array(array(uri => media_type), ...)
+     * @param string $cssMediaType
+     * @param int|null $offset
+     * @param bool $checkPath
+     */
+    public function addCSS(
+        $cssUri,
+        $cssMediaType = 'all',
+        $offset = null,
+        $checkPath = true): void
+    {
+        if (null === $this->getContext()->controller) {
+            throw new \RuntimeException('No context controller available');
```

Comment:
```suggestion
            throw new RuntimeException('No context controller available');
```

Suggestion: With appropriate `use` statement